controllers/v1/questionsController.js

```diff
@@ -685,4 +685,586 @@ module.exports = class Questions extends Abstract {
     });
   }
 
+
+  /**
+  * @api {post} /assessment/api/v1/questions/bulkCreateSurveyQuestions?solutionExternalId=:solutionExternalId Upload survey questions CSV
+  * @apiVersion 1.0.0
+  * @apiName Upload survey questions via csv.
+  * @apiGroup Questions
+  * @apiSampleRequest /assessment/api/v1/questions/bulkCreateSurveyQuestions?solutionExternalId=Test-survey
+  * @apiParam {File} questions Mandatory questions file of type CSV.
+  * @apiUse successBody
+  * @apiUse errorBody
+  */
+
+  /**
+   * Bulk create survey questions.
+   * @method
+   * @name bulkCreateSurveyQuestions
+   * @param {Object} req - requested data.
+   * @param {Object} req.files.questions - questions csv data. 
+   * @returns {CSV} - Same existing csv with extra field _SYSTEM_ID to indicate 
+   * whether the question is created or not . If created question id will be provided. 
+   */
+
+  bulkCreateSurveyQuestions(req) {
```

Comment:
@deepakr123 - Suggest merging it with existing API.

Reply:
resolved

Reply:
resolved